home/screens/BranchDetailsScreen/BranchDetailsView.tsx

```diff
@@ -68,14 +68,16 @@ export function BranchDetailsView({ error, data, refetch, branchName, networkSta
         ListHeaderComponent={<SectionHeader header="Updates" style={{ paddingTop: 0 }} />}
         keyExtractor={(update) => update.id}
         contentContainerStyle={{ padding: spacing[4] }}
-        ItemSeparatorComponent={() => <Spacer.Vertical size="small" />}
+        ItemSeparatorComponent={() => <Divider style={{ height: 1 }} />}
         ListEmptyComponent={() => <EmptySection />}
-        renderItem={({ item: update }) => (
+        renderItem={({ item: update, index }) => (
           <UpdateListItem
             id={update.id}
             message={update.message ?? undefined}
             manifestPermalink={update.manifestPermalink}
             createdAt={update.createdAt}
+            first={index === 0}
+            last={index === (data.app.byId.updateBranchByName?.updates ?? []).length - 1}
```

Comment:
looks great - the only thing I would be sure to verify is that this still works when the length of the list changes, you might have to pass in length as an `extraProp` to flatlist if it doesn't